### PR TITLE
Important qube template update 1.0.2 released!

### PR DESCRIPTION
**Pull request by JohnnyQ5**
A new release of the main template in qube has just been released. This automated pull-request attempts to apply the relevant updates to this Project.

Please make sure to merge this pull-request as soon as possible. Once complete, make a new minor release of your Project.

For more information on the actual changes, read the latest qube changelog.

### CODE REVIEW

**Comment by sven1103** [comment]
@JohnnyQ5 thank you for your contribution, but you need to open the PR to `development` ...

File: .qube.yml [hunk 1]
```diff
@@ -8,13 +8,13 @@ domain: lib
...
 project_slug_no_hyphen: data_model_lib
-template_version: '1.0.0 # <<QUBE_NO_BUMP>>'
+template_version: '1.0.3 # <<QUBE_NO_BUMP>>'
 template_handle: lib-java
 github_username: qbicsoftware
 creator_github_username: sven1103
-is_github_repo: True
-is_repo_private: False
-is_github_orga: True
+is_github_repo: true
```
Comment (Zethson) [inline .qube.yml:15]: They should be uppercase now, right?

Comment (sven1103) [reply .qube.yml:15]: exactly :)

File: docs/usage.rst [hunk 2]
```diff
@@ -2,6 +2,6 @@
 Usage
 =====
 
-To use Data_models in a project::
```
Comment (Zethson) [inline docs/usage.rst:5]: Ooops.

This is a relict from a Python template copy pasta haha.

The usage section should look different.
Has to be fixed in the template though and not in this PR.

**Review by Zethson — APPROVED** [review]
@JohnnyQ5 <3

**Comment by sven1103** [comment]
[Utilise JohnnyQ5 bot for all automated work](https://trello.com/c/jIq1iyig/345-utilise-johnnyq5-bot-for-all-automated-work)

**Comment by sven1103** [comment]
[Get sync working from template](https://trello.com/c/0Vd2xxuR/346-get-sync-working-from-template)

**Pull request merged**